### ISSUE TITLE
pyproxy.gen.ts:237 Uncaught Error

### ISSUE THREAD

**Issue by Python-Ninja-Hebi** [pyscript/pyscript#520]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help

### What happened?

I used the example [todo-pylist.html](https://github.com/pyscript/pyscript/blob/main/examples/todo-pylist.html)

and got the error

pyproxy.gen.ts:237 Uncaught Error: This borrowed proxy was automatically destroyed at the end of a function call. Try using create_proxy or create_once_callable.
The object was of type "function" and had repr "<function on_click_new_task_btn at 0xe25e00>"
    at _getPtr (pyproxy.gen.ts:237:11)
    at PyProxyClass.apply (pyproxy.gen.ts:1145:32)
    at Object.apply (pyproxy.gen.ts:1022:18)

### What browsers are you seeing the problem on? (if applicable)

Chrome, Safari

### Console info

```shell
pyproxy.gen.ts:237 Uncaught Error: This borrowed proxy was automatically destroyed at the end of a function call. Try using create_proxy or create_once_callable.
The object was of type "function" and had repr "<function on_click_new_task_btn at 0xe25e00>"
    at _getPtr (pyproxy.gen.ts:237:11)
    at PyProxyClass.apply (pyproxy.gen.ts:1145:32)
    at Object.apply (pyproxy.gen.ts:1022:18)
```

**Comment by JeffersGlass** [comment]
I'm currently seeing the same error when using the js bundle at `https://pyscript.net/alpha/pyscript.js`, but **not** when building pyscript locally... there are [some differences](https://www.diffchecker.com/jI9CeNcv) between at least my local build (left in the diffchecker) and the build at pyscript.net, but to my casual eye it seems like they're mostly related to livereload or Svetle compilation options, which makes sense for a production vs dev build...

Maybe someone with a better view of the build-and-deploy workflow can help out here.

**Comment by aqemery** [comment]
issues was resolved here: https://github.com/pyscript/pyscript/issues/485 however it is still an issue on the alpha release

**Comment by Python-Ninja-Hebi** [comment]
When will it be released?

**Comment by JeffersGlass** [comment]
This issue is confirmed to be resolved as @aqemery noted, and is still resolved as of 2022.09.1-RC2
